assets/js/modules/analytics/components/setup/SetupFormGA4.stories.js

```diff
@@ -1,7 +1,7 @@
 /**
  * SetupFormGA4 component stories.
  *
- * Site Kit by Google, Copyright 2021 Google LLC
+ * Site Kit by Google, Copyright 2023 Google LLC
```

Comment:
```suggestion
 * Site Kit by Google, Copyright 2021 Google LLC
```

Same here, let's leave this as-it-was.

Reply:
Updated.